src/applications/appeals/shared/constants.js

```diff
@@ -88,8 +95,16 @@ export const MAX_LENGTH = {
   COUNTRY: 2,
```

Comment:
SC also has this named `ADDRESS_COUNTRY` because it was easy to confuse the phone & address country constants. So please rename this to `ADDRESS_COUNTRY` and update HLR & NOD code to match. Thanks!

Reply:
Hmmm, We seem to be missing the `internationalPostalCode` max length of `16` - I think profile limits the input of that code so it's not a high-priority fix; but, we don't limit it in the submit transformer before submission which we should.